<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StaffSideView</class>
 <widget class="QDialog" name="StaffSideView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#F0F2F5;
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="logo_lable">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>200</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="images.qrc">:/asset/loginpageAsset/FITLogo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="topMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="greeting_label">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Choose School Year</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:#FFFFFF;
border-radius: 10px;
background-color: #FFFFFF</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <property name="leftMargin">
               <number>20</number>
              </property>
              <property name="topMargin">
               <number>15</number>
              </property>
              <property name="rightMargin">
               <number>20</number>
              </property>
              <property name="bottomMargin">
               <number>15</number>
              </property>
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QComboBox" name="school_year_box">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>50</height>
                   </size>
                  </property>
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">/*
QComboBox { 
	border: 1px solid rgb(212, 212, 212); 
	border-radius: 25px; 
	color: black; 
	padding-left: 15px; 
}

QComboBox:focus {
    color: black;
	border: 1px solid blue; 
}

QComboBox::down-arrow { 
	image: url(qrc:/Asset/staffViewAsset/arrow-down-3101.svg);
} */
</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>293</width>
                    <height>15</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QPushButton" name="confirm_button">
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
    background-color: #42b72a;
    border-width: 2px;
    border-radius: 15px;
    border-color: #42b72a;
	color: white;
}

QPushButton:hover {
    background-color: #37a520;
    border-color: #37a520;
	color: white;
}</string>
                    </property>
                    <property name="text">
                     <string>OK</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Fixed</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>30</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QPushButton" name="add_school_year_button">
                    <property name="minimumSize">
                     <size>
                      <width>150</width>
                      <height>50</height>
                     </size>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QPushButton {
    background-color: #0866FF;
    border-width: 2px;
    border-radius: 15px;
    border-color: #0866FF;
	color: white;
}

QPushButton:hover {
    background-color: #038EFF;
    border-color: #038EFF;
	color: white;
}</string>
                    </property>
                    <property name="text">
                     <string>Add </string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="changePasswordButton">
           <property name="text">
            <string>Change password</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
